martin/src/pg/query_tables.rs

```diff
@@ -171,19 +176,34 @@ pub async fn table_to_query(
 
     let extent = info.extent.unwrap_or(DEFAULT_EXTENT);
     let buffer = info.buffer.unwrap_or(DEFAULT_BUFFER);
+    let margin = f64::from(buffer) / f64::from(extent);
+    let proj = if let Some(proj) = &info.proj {
+        proj
+    } else {
+        ""
+    };
+    let proj_unit = if let Some(proj_unit) = &info.proj_unit {
+        proj_unit
+    } else {
+        ""
+    };
```

Comment:
nit:
this here seems a bit unconventional.
Would below work?

If we do fold the `None` case anyhow, we might as well not have it and change this to be `String` instead, or am I missing something?

```suggestion
    let proj = &info.proj.unwrap_or_default();
    let proj_unit = &info.proj_unit.unwrap_or_default();
```

Reply:
I am by no means a Rustacean (in fact this is the first time I've written it) so if there is a better or more idiomatic way here, I'd be happy to change it. This was more following the pattern established above by the other conditional assignments. The expansion of the more succinct ternary-like syntax was applied by the linter.

Reply:
I should note that I'm primarily holding off on additional changes here at the moment until we have at least agreed on the overall approach being reasonable.